src/index.html

```diff
@@ -4,9 +4,448 @@
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div
+          class="container__card card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__img"
+          >
+
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__subtittle">Product code: 195434</p>
+
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+
+            <p class="card__review-number">
+              Reviews: 5
+            </p>
+          </div>
+
+
+          <div class="card__price-container">
+            <p class="card__price">Price:</p>
+            <p class="card__cost ">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
+        </div>
+
+        <div
+          class="container__card card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__img"
+          >
+
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__subtittle">Product code: 195434</p>
+
+
+          <div class="card__review-container">
+            <div class="stars stars--3">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+
+            <p class="card__review-number">
+              Reviews: 5
+            </p>
+          </div>
+
+
+          <div class="card__price-container">
+            <p class="card__price">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+          >
```

Comment:
When there are 1 or 2 attributes, it is not necessary to transfer them to new lines, this should be done only when there are 3 or more attributes. Need to fix it everywhere
```suggestion
          <a href="#" class="card__buy">
```